src/plone/restapi/tests/test_services_transactions.py

```diff
@@ -0,0 +1,50 @@
+from plone.app.testing import setRoles
+from plone.app.testing import SITE_OWNER_NAME
+from plone.app.testing import SITE_OWNER_PASSWORD
+from plone.app.testing import TEST_USER_ID
+from plone.restapi.testing import PLONE_RESTAPI_DX_FUNCTIONAL_TESTING
+from plone.restapi.testing import RelativeSession
+
+import transaction
+import unittest
+
+
+class TestTransactionsEndpoint(unittest.TestCase):
+    layer = PLONE_RESTAPI_DX_FUNCTIONAL_TESTING
+
+    def setUp(self):
+        self.app = self.layer["app"]
+        self.portal = self.layer["portal"]
+        self.portal_url = self.portal.absolute_url()
+        setRoles(self.portal, TEST_USER_ID, ["Manager"])
+
+        self.api_session = RelativeSession(self.portal_url, test=self)
+        self.api_session.headers.update({"Accept": "application/json"})
+        self.api_session.auth = (SITE_OWNER_NAME, SITE_OWNER_PASSWORD)
+
+        self.endpoint_url = f"{self.portal_url}/@transactions"
+
+        transaction.commit()
+
+    def tearDown(self):
+        self.api_session.close()
+
+    def test_get_datastructure(self):
+        response = self.api_session.get(self.endpoint_url)
+        data = response.json()
+
+        keys = ["description", "id", "size", "time", "user_name"]
```

Comment:
This should also be "username" and not "user_name"

Reply:
@robgietema  Are you sure? because `user_name` is present in each object of transactions not `username` and changing `user_name` to `username` might cause test failure as test is checking coming objects elements from `@transactions` endpoint like `description`, `id`, `size`, `time`, `user_name`.

Reply:
@MdSahil-oss the variable which is returned via the restapi should be "username" so it is consistant with the rest of the restapi. Internal variables can stay "user_name".

Reply:
@robgietema changes have been made. Now, `user_name` is `username` to make naming convention consistent.